other/field-types/string.md

```diff
@@ -1,6 +1,6 @@
 # `string`
 
-`string` adds an editable text string field to the schema. Setting `textarea: true` presents an interface that allows multiple lines. You may set minimum and maximum numbers of characters using the `min` and `max` options.
+`string` adds an editable text string field to the schema.
```

Comment:
Simply "text field" so as to not define with the word.

Reply:
Updated.